view/adminhtml/templates/configuration.phtml

```diff
@@ -62,6 +62,11 @@ document.addEventListener("DOMContentLoaded", function(event) {
 					$('#algoliasearch_cc_analytics_cc_analytics_group table').hide();
 				}
 
+				// Hide personalization config if not available
+				if (personalizationStatus < 2) {
```

Comment:
from a JS good practice perspective, JS developers prefer explicit the meaning of the constant:
```js
var PERSONALIZATION_NON_AVAILABLE_STATUS = 0;
var PERSONALIZATION_AVAILABLE_NOT_ENABLED_STATUS = 1
if (personalizationStatus === PERSONALIZATION_NON_AVAILABLE_STATUS || PERSONALIZATION_AVAILABLE_NOT_ENABLED_STATUS) { 
   ...
 }
```

Even better if we expose the constant from the backend to the frontend in one common place (it's more searchable in the codebase)

Reply:
I think we'll keep as is as it's for the backend side. This status check isn't used anywhere else at the moment. Still great insights for us to know about constant usages in JS!